gravitee-apim-integration-tests/src/test/java/io/gravitee/apim/integration/tests/messages/AbstractKafkaEndpointIntegrationTest.java

```diff
@@ -115,6 +124,25 @@ private static void createTopic(final AdminClient adminClient) throws Interrupte
         adminClient.createTopics(topics).all().get(30, TimeUnit.SECONDS);
     }
 
+    protected static KafkaConsumer<String, byte[]> getKafkaConsumer(Vertx vertx) {
+        Map<String, String> config = new HashMap<>();
+        config.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, kafka.getBootstrapServers());
+        config.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class.getName());
+        config.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, ByteArrayDeserializer.class.getName());
+        config.put(ConsumerConfig.GROUP_ID_CONFIG, UUID.randomUUID().toString());
+        config.put(ConsumerConfig.CLIENT_ID_CONFIG, UUID.randomUUID().toString());
+        config.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
+        return KafkaConsumer.create(vertx, config);
+    }
+
+    protected static Flowable<KafkaConsumerRecord<String, byte[]>> subscribeMessages(KafkaConsumer<String, byte[]> kafkaConsumer) {
```

Comment:
Only my opinion, but the notion of message is related to the object we manipulate inside the gateway when here you consumes kafka record, I propose two alternatives:
```suggestion
    protected static Flowable<KafkaConsumerRecord<String, byte[]>> subscribeToKafka(KafkaConsumer<String, byte[]> kafkaConsumer) {
```
or
```suggestion
    protected static Flowable<KafkaConsumerRecord<String, byte[]>> consumeKafkaRecords(KafkaConsumer<String, byte[]> kafkaConsumer) {
```